core/devmode-spi/pom.xml

```diff
@@ -13,6 +13,18 @@
     <artifactId>quarkus-development-mode-spi</artifactId>
     <name>Quarkus - Development mode - SPI</name>
     <description>SPI classes for Quarkus Development mode.</description>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>9</source>
+                    <target>9</target>
+                </configuration>
```

Comment:
Do we need this?

Reply:
Oops, intellij must have added it while I was fighting with it in windows.